robotouille_wrapper.py

```diff
@@ -141,11 +153,23 @@ def _handle_action(self, action):
             else:
                 item_status["cook"]["cooking"] = True
             return self.prev_step
+        elif action_name == "fry" or action_name == "fry_cut_item":
```

Comment:
Can you update this function's documentation (since it has a representation of the state)